x-pack/platform/plugins/shared/alerting/common/routes/schedule/constants.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const ISO_DATE_REGEX = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{3}Z$/;
+export const WEEKDAY_REGEX = '^(((\\+|-)[1-4])?(MO|TU|WE|TH|FR|SA|SU))$';
+export const DURATION_REGEX = /(\d+)(s|m|h)/;
```

Comment:
I feel like just these 3 is a bit limiting. as a user, I wouldn't want to have to think how many hours a month is.

Reply:
Is the `m` for minutes or months? Usually, maintenance windows take a couple of minutes to a couple of hours. We could add the `d` for days if you feel we need it.

Reply:
`m` is for minutes.

Reply:
[added](https://github.com/elastic/kibana/pull/210584/commits/190edbbd93ec163b235703f155bd4d7099cf8022)